spark-connector/spark-common/src/test/java/org/apache/gravitino/spark/connector/integration/test/SparkCommonIT.java

```diff
@@ -119,6 +119,14 @@ private static String getRowLevelDeleteTableSql(
 
   protected abstract boolean supportsReplaceColumns();
 
+  protected abstract boolean supportsProperties();
```

Comment:
it's a little hard to understand `supportsProperties` here, could you use a better name and add some comment?

Reply:
In `JdbcDatabaseOperations`, an exception message "Properties are not supported yet" will be thrown, indicating that the operation of the jdbc object does not support to use properties, so I think supportsProperties is not hard to understand

Reply:
How about `supportsSchemaProperties`?

Reply:
Neither the schema nor the table of jdbc supports user custom properties. Although the operation of table properties will not throw exception, it will not actually take effect. Or do you think that the schema and table should be distinguished?

Reply:
got it, how about `supportsSchemaAndTableProperties`?

Reply:
ok, it's good!